internal/command/jsonformat/computed/renderers/sensitive_block.go

```diff
@@ -37,15 +37,9 @@ func (renderer sensitiveBlockRenderer) WarningsHuman(diff computed.Diff, indent
 		return []string{}
 	}
 
-	var warning string
 	if renderer.beforeSensitive {
-		warning = fmt.Sprintf("  # [yellow]Warning[reset]: this block will no longer be marked as sensitive\n%s  # after applying this change.", formatIndent(indent))
+		return []string{fmt.Sprintf("  # [yellow]Warning[reset]: this block will no longer be marked as sensitive\n%s  # after applying this change.", formatIndent(indent))}
```

Comment:
Noted elsewhere in an invisible comment, but for clarity again: we need to colorize these format strings before we interpolate them, not later.

Reply:
I think this should be fixed in this PR: https://github.com/hashicorp/terraform/pull/32502